generators/techs/templates/src/app/techs/Techs.vue

```diff
@@ -4,7 +4,7 @@
       Cooked with all these awesome technologies:
     </h2>
     <div class="techs">
-      <tech v-for="tech in techs" :tech="tech"></tech>
+      <tech v-for="tech in techs" :tech="tech" :key="item.key"></tech>
```

Comment:
use tech.key instead, item.key fails to load for me, though it does get rid of the error.